src/scripting/lua_game.cc

```diff
@@ -1248,9 +1251,9 @@ int LuaObjective::set_visible(lua_State* L) {
    .. attribute:: done
 
       (RW) defines if this objective is already fulfilled. If done is
-      :const`true`, the objective will not be shown to the user, no matter what.
+      `true`, the objective will not be shown to the user, no matter what.
       :attr:`visible` is set to. A savegame will be created when this attribute
-      is changed to :const`true`.
+      is changed to `true`.
```

Comment:
use double backticks for all constants so they are shown ``in code style`` and not *in italics*

Reply:
There were more occurrences like this, changed them all.

There should be general rules for the documentation, imho. Will start to make a wiki article to archive this.